pkg/components/exporter/exporter_test.go

```diff
@@ -0,0 +1,75 @@
+package exporter
+
+import (
+	"testing"
+
+	ko "github.com/sustainable.computing.io/kepler-operator/pkg/api/v1alpha1"
+	"gotest.tools/v3/assert"
+	appsv1 "k8s.io/api/apps/v1"
+	corev1 "k8s.io/api/core/v1"
+)
+
+func NodeSelectorFromDS(ds *appsv1.DaemonSet) map[string]string {
+	return ds.Spec.Template.Spec.NodeSelector
+}
+
+func TolerationsFromDS(ds *appsv1.DaemonSet) []corev1.Toleration {
+	return ds.Spec.Template.Spec.Tolerations
+}
+
+func TestNodeSelection(t *testing.T) {
+
+	tt := []struct {
+		spec     ko.ExporterSpec
```

Comment:
could you please add a scenario? 
```suggestion
		scenario   string
		spec     ko.ExporterSpec
```
And use 
```
t.Run(tc.scenario, func(...){
   t.Parallel()

})
```